data/virt_autotest/host_unattended_installation_files/combustion/script.x86_64.ttyS1

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+# combustion: network
+
+grub2_config() {
+    file=/etc/default/grub
+    sed -i 's/^\s*GRUB_TIMEOUT=.*$/GRUB_TIMEOUT=15/g' $file
+    sed -i -r 's/^(\s|#)*GRUB_TERMINAL=.*$/GRUB_TERMINAL="gfxterm console"/g' $file
+    sed -i -r 's/^(\s|#)*GRUB_TERMINAL_INPUT=.*$/GRUB_TERMINAL_INPUT="gfxterm console"/g' $file
+    sed -i -r 's/^(\s|#)*GRUB_TERMINAL_OUTPUT=.*$/GRUB_TERMINAL_OUTPUT="gfxterm console"/g' $file
+    sed -i 's/console=tty0//g' $file
+    sed -i 's/console=tty[^ ]*/console=ttyS1,115200/g' $file
```

Comment:
same here.
Next step, we will transfer more x86 HWs to uefi and used for sl micro test. Among them, serial console are not the same.

Reply:
First boot configuration file provided and used directly on URL. It is not configurable and it is not necessary. Then you create corresponding files for other serial consoles.

Reply:
Done